src/components/screens/ServiceTile.tsx

```diff
@@ -21,6 +21,7 @@ export type TServiceTile = {
   params?: any;
   query?: string;
   routeName: string;
+  svg?: string;
```

Comment:
where does this svg came from and what is its purpose? i could not find any hint in the commit message or PR description.

Reply:
[9fa123f](https://github.com/smart-village-solutions/smart-village-app-app/commit/9fa123f31738e43ad010657b72c7bd4219f63428)